src/lib/viewers/controls/color-picker/ColorPickerPalette.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ControlsBar from '../controls-bar';
+import './ColorPickerPalette.scss';
+
+export type Props = {
+    onColorSelect: () => void; // TODO: this method wil need to take in a color string param
+};
+
+export default function ColorPickerPalette({ onColorSelect }: Props): JSX.Element {
+    const colors = ['#0061d5', '#26c281', '#ed3757', '#f5b31b', '#ffd700', '#4826c2'];
+
+    return (
+        <ControlsBar>
```

Comment:
This component is already inside a `ControlsBar`. Is this needed?

Reply:
Yes it looks like it's needed. When I remove the ControlsBar wrapper component, the `background: inherit` style is not being applied properly.

Reply:
We should avoid nesting components solely for visual styling. If we need a particular background color, let's add a style for it to the `bp-ColorPickerPalette` element.

Reply:
That makes sense, will fix.